# Master MAVLink bridge params
master_sys_id: 255
master_udp_link: "127.0.0.1:30000"
master_serial_link: "/dev/ttyUSB0"
ENABLE_MASTER_GCS: True
USE_SERIAL: True
CONN_TOUT: 5.0

# Robot MAVLink bridge params
robot_udp_link: "127.0.0.1:30000"
robot_serial_link: "/dev/ttyTelem"

# Common params
serial_baudrate: 57600
